example-config.yml

```diff
@@ -60,13 +58,36 @@ persistence:
 #    apiBaseUrl: https://otp-middleware.example.com
 #    apiKey: your-middleware-api-key
 
+### Adding additional menu items to the main menu items. Use the separator flag
+### to include a separator line if you have groups of menu items
+### If a Top level menu item contains submenu items (children) then use the 'children' flag.
+### Icon URL is preferable over iconType. If non are given then a 'bus' iconType is used.
+#App menu
+#extraMenuItems:
+#  - id: link-list
+#    label: List of Links
```

Comment:
Apologies, I just realized that some of these links need to be localizable too 😵 but not all.
One way to accomplish that could be to write:
```yaml
extraMenuItems:
  - id: link-test
    label:
      en-US: List of Links
      fr-FR: Liste de liens
  - id: brand-link-test
    label: Brand/Product Name # Default top-level text for brand names that shouldn't be localized.
```

Reply:
I didn't like the idea of localized strings living in multiple places, so I added a way to set labels in the config file based on ids. (7c35a11b0b92e6748f84ebe8ebdcc01dc8807371) What do you think?

Reply:
Your approach should work too, but add a comment regarding the prefixing that the implementer should use to override the strings (also add an example in `example-config.yml`)